tests/html/click-tests.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <title>Gamepad Navigator Tabbing Tests</title>
```

Comment:
This title copied from the other test should be updated.  (I do this all the time myself, it's not a big deal.)